cmd/triggerRun/testdata/triggerBinding.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
filename should be something like this `cmd/triggerRun/testdata/triggerbinding.yaml`